Keyed/Letters.xml

```diff
@@ -157,13 +157,13 @@
   <!-- EN: The caravan is selling people who have relationships with your {0} or prisoners: -->
   <LetterRelatedPawnsTradingWithOtherCaravan>Karawana sprzedaje ludzi, którzy są związani z Twoimi kolonistami lub więźniami:</LetterRelatedPawnsTradingWithOtherCaravan>
   <!-- EN: There are people who have relationships with your {0} or prisoners in the group: -->
-  <LetterRelatedPawnsGroupGeneric>TODO</LetterRelatedPawnsGroupGeneric>
+  <LetterRelatedPawnsGroupGeneric>W przybywającej grupie znajdują się osoby, które są związane z kolonistami lub więźniami:</LetterRelatedPawnsGroupGeneric>
```

Comment:
czemu nie jest użyte koloniści zamiast {0} ?? jest kilka takich przykładów

Reply:
Czemu nie jest, czy czemu jest?
Związane z {0=koloniści} lub więźniami. 
Dlatego jest zamiast {0} jest **kolonistami** 

Ale pewny nie jestem czy {0} odpowiada tylko za to jedno słowo. Tak czy inaczej przy użyciu {0} będzie wyświetlać błędnie

Reply:
Myślę że można zmienić tak jak proponujesz, kiedyś siądę i spróbuje kod zdekompilować

Reply:
<PlayerColony.pawnsPlural>koloniści</PlayerColony.pawnsPlural> odpowiada za {0}

Reply:
Prawdopodobnie PlayerColony.pawnsPlural występuje w tłumaczeniach:

LetterRelatedPawnsNeutralGroup argument {0}
NoColonistWithConstructionSkillTip argument {0}
AllColonistsWithConstructionSkillHaveDisaledConstructingTip argument {0}
Z DefInjected\IncidentDef argument {1}
LetterAmbushInExistingMap argument {0}
LetterRelatedPawnsGroupGeneric argument {0}
LetterRelatedPawnsRaidEnemy argument {0}
LetterRelatedPawnsNeutralGroup argument {0}

Reply:
Czyli nie tylko koloniści, ale i tubylcy moga sie pojawić

Reply:
w grze obiekt ten nazywa się Faction.OfPlayer.def.pawnsPlural.
Jest on brany z od aktualnego gracza. Jeśli można grać jako tubylec to będą też i tubylcy, ale chyba tylko jako kolonista można grać